flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/runtime/batch/table/CalcITCase.scala

```diff
@@ -593,6 +593,27 @@ class CalcITCase(
     val results = filterDs.toDataSet[Row].collect()
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
+
+  @Test
+  def testColumnOperations(): Unit = {
```

Comment:
I think we can remove the ITCase, due to the column operation only the Table API level Features， do not touch runtime logic. What do you think?